ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -290,6 +290,17 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-users-static
+- as: nutanix-ipi-zones-single-f28
+  cluster: build01
+  cron: 27 16 16 * *
+  steps:
+    cluster_profile: nutanix-qe-zone
+    env:
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      SINGLE_ZONE: "true"
+    test:
+    - chain: openshift-e2e-test-qe
```

Comment:
If this job is only for installation test cases, then we do not need to run e2e testing, but use `ref: cucushift-installer-check-cluster-health` to replace it. If you wan to run it with e2e testing, I think we should move it to regular ci yaml.